Core/OfficeDevPnP.Core/Framework/Provisioning/Connectors/OpenXML/PnPPackageHelper.cs

```diff
@@ -13,33 +12,33 @@ namespace OfficeDevPnP.Core.Framework.Provisioning.Connectors.OpenXML
     /// </summary>
     public static class PnPPackageExtensions
     {
-        public static Byte[] PackTemplate(this PnPInfo pnpInfo)
+        public static MemoryStream PackTemplate(this PnPInfo pnpInfo)
```

Comment:
Any preference on method name as I cannot do a simple overload as they both have the same signature. PackTemplateAsStream?